lib/hsa/kernels/frb_transpose_amd.cl

```diff
@@ -0,0 +1,116 @@
+#define T 16 //(sgx/sgy)
+#define BS T*gs
+
+#define lx get_local_id(0)
+#define ly get_local_id(1)
+
+#define gx get_group_id(0)
+#define gy get_group_id(1)
+
+#define sgx get_local_size(0)
+#define sgy get_local_size(1)
+
+#define ggx get_global_size(0)
+#define ggy get_global_size(1)
+
+#define ngx get_num_groups(0)
+#define ngy get_num_groups(1)
+
+//input dimensions [n_samples, n_pol, n_beams] -> [N,2048]
+//output dimensions [n_pol, n_beams, n_samples] -> [2048,N]
+
+#define flip(sel, a, b, c) \
+        c = select(b,a,sel);
+
+#define flop(sel, a, b, c) \
+        a = select(a,c,sel); \
+        b = select(c,b,sel);
+
+
+__kernel
+//__attribute__((reqd_work_group_size(32, 2, 1)))
+//__attribute__((amdgpu_num_vgpr(28)))
+void transpose(__global uint *input, __global uint *output) {
```

Comment:
This is doing lots of cool things, but it makes it harder to follow what the actual operation is (compared with `transpose.cl`), could you add some comments to describe the transpose in simple (pseudocode?) terms.

Reply:
I think for now we just remove this file from the merge. It's not being used.